File: app/src/main/java/com/mediaplayer/components/MessageAlert.java
package com.mediaplayer.components;


import android.content.Context;
import android.os.Handler;
import android.widget.Toast;

public class MessageAlert {

    public void showToast(final String msg, final Context ctx) {
        Handler handler = new Handler(ctx.getMainLooper());
        handler.post(new Runnable() {
            public void run() {
                Toast.makeText(ctx, msg, Toast.LENGTH_LONG).show();
            }
        });
    }

}

File: app/src/main/java/com/mediaplayer/components/MessageAlert.kt
package com.mediaplayer.components


import android.content.Context
import android.os.Handler
import android.widget.Toast

class MessageAlert {

    fun showToast(msg: String, ctx: Context) {
        val handler = Handler(ctx.mainLooper)
        handler.post { Toast.makeText(ctx, msg, Toast.LENGTH_LONG).show() }
    }

}
